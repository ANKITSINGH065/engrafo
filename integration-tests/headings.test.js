var utils = require("./utils");

test("bold is stripped from title", done => {
  utils.expectBodyToMatchSnapshot("headings/bold-title.tex", done);
});

test("TITLES IN ALL CAPS are fixed", done => {
  utils.expectBodyToMatchSnapshot("headings/all-caps-titles.tex", done);
});
