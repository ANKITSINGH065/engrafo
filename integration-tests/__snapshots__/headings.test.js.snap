// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TITLES IN ALL CAPS are fixed 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    

    <dt-article>
      

      <h1>
        Why Am I Shouting
      </h1>
      

      <div
        class="engrafo-metadata"
      >
        
  
        <div
          class="authors"
        >
          
    
        </div>
        
  
        <div
          class="engrafo-metadata-custom"
        />
        

      </div>
      


      <h2
        id="section-1"
      >
        <span
          class="section-number"
        >
          <a
            href="#section-1"
          >
            1
          </a>
        </span>
        Urgh Stop It
      </h2>
      

      <p>
        Hello.
      </p>
      

    </dt-article>
    

    <dt-appendix>
      

      


      <div
        class="l-body"
      />
      

    </dt-appendix>
    



    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;

exports[`bold is stripped from title 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    

    <dt-article>
      

      <h1>
         Why would someone do this
      </h1>
      

      <div
        class="engrafo-metadata"
      >
        
  
        <div
          class="authors"
        >
          
    
        </div>
        
  
        <div
          class="engrafo-metadata-custom"
        />
        

      </div>
      


      <p>
        Hello.
      </p>
      

    </dt-article>
    

    <dt-appendix>
      

      


      <div
        class="l-body"
      />
      

    </dt-appendix>
    



    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;
